djhtml/modes.py

```diff
@@ -385,10 +385,10 @@ class DjJS(DjTXT):
         r"//.*",
         r"/\*",
         r"[$\w-]+:",
-        r'"(?:\\.|[^\\"])*"',  # "string"
-        r"'(?:\\.|[^\\'])*'",  # 'string'
-        r"`(?:\\.|[^\\`])*`",  # `string`
-        r"/(?:\\.|[^\\/ \n])*/",  # /string/ (but without spaces)
+        r'"(?:\\.|[^"])*"',  # "string"
```

Comment:
Note that this will now also erroneously match `"\"`, with `\` matching the `[^"]` branch of the conditional.

Reply:
`"\"` is a syntax error in JavaScript:

```js
var str = "\"
Uncaught SyntaxError: Invalid or unexpected token
```